src/build-task/AugurkCSharpAnalyzer/task.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "3b04573e-7699-440a-93ae-8f7b06259078",
+  "name": "AugurkCSharpAnalyzer",
+  "friendlyName": "Run Augurk C# Analysis",
+  "description": "Performs a static code analysis over your test automation and uploads the results to Augurk",
+  "category": "Utility",
+  "author": "Augurk",
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 2
+  },
+  "demands": [
+    "augurk-csharpanalyzer"
+  ],
+  "instanceNameFormat": "Analyze $(solution) and upload to Augurk",
+  "groups": [
+    {
+      "name": "advanced",
+      "displayName": "Advanced",
+      "isExpanded": false
+    }
+  ],
+  "inputs": [
+    {
+      "name": "solution",
+      "type": "filePath",
+      "label": "Solution(s)",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "Path to the solution file that should be analyzed"
+    },
+    {
+      "name": "specificationProject",
+      "type": "string",
+      "label": "Specification project",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "Name of the project containing the feature files"
```

Comment:
Perhaps we should indicate that this should be a testproject containing the SpecFlow test automation instead?